tests/test_preprocess.py

```diff
@@ -50,43 +54,28 @@ def test_preprocess_one_feature():
     finally:
         rmtree(output_directory)
 
-def test_preprocess_all_features():
+
+def test_preprocess_single_feature():
     """
-    Tests preprocessing several PDB files into their features representation HDF5 file.
+    Tests preprocessing for single feature.
     """
 
-    output_directory = mkdtemp()
-
-    prefix = os.path.join(output_directory, "test-preprocess")
+    imp = importlib.import_module(('deeprankcore.feature.sasa'))    # linting doesn't like: from deeprankcore.feature import sasa below top_level
```

Comment:
You could import sasa from outside the function (very top of the script where are the other modules imports) and then just do `preprocess_tester([sasa])`

Reply:
I know, but preferred it this way because it's consistent with what I do in the 2nd test. I forgot to remove that comment, it was a note to self more than anything else.

Reply:
Okay, both are fine :)